packages/maskbook/src/plugins/Tweet/define.tsx

```diff
@@ -0,0 +1,18 @@
+import { PluginConfig, PluginStage, PluginScope } from '../types'
+import { PLUGIN_IDENTIFIER } from './constants'
+import { usePostInfoDetails } from '../../components/DataSource/usePostInfo'
+import TweetDialog from './UI/TweetDialog'
+
+export const TweetPluginDefine: PluginConfig = {
+    pluginName: 'Tweet',
```

Comment:
Please give this plugin a more intuitive name (and rename the folder) thanks

Reply:
Done.
1. Renamed plugin from Tweet to VCent.
2. Renamed TweetDialog to VCentDialog.
3. Renamed folder to VCent.